src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaStudentSupportCallResponseService.java

```diff
@@ -101,7 +101,7 @@ public void sendNotificationsByWorkgroupId(Long workgroupId) {
 
         for (Schedule schedule : workgroup.getSchedules()) {
             // Ignore historical schedules
-            if (schedule.getYear() >= currentYear) {
+            if (currentYear <= (schedule.getYear() + 1)) {
```

Comment:
Instead of repeating this logic, what about an isHistorical() method on Schedule?